README.md

```diff
@@ -65,12 +66,20 @@ In order to make use of this software you must install the following software on
  * Ubuntu Bionic (18.04)
  * ROS Melodic Morenia
 
-Installation instructions for ROS Melodic on Ubuntu Bionic may be found [here](http://wiki.ros.org/melodic/Installation/Ubuntu).
+or
+
+ * Ubuntu Xenial Xerus (16.04)
+ * ROS Kinetic Kame
+
+Basic RxROS functionality requires only installation of the ROS "ROS-Base" packages,
+where as full RxROS functionality, which includes transformation (tf) observers
+and operations, requires installation of the ROS "Desktop-Full" packages (Recommended).
```

Comment:
More of a general comment: please do not hard-wrap lines, and keep single-sentence-per-line. It makes diffs much easier to read and supports reflowing the text better.